frontend/components/markup-modal/copied-content-indicator.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import posed from 'react-pose'
+import styled from 'styled-components'
+
+const ContentAnchor = styled.span`
+  position: relative;
+  display: inline-block;
+`
+
+// 100vw allows providing styled content which is wider than its container.
+const ContentContainer = styled.span`
+  width: 100vw;
+
+  position: absolute;
+  left: 50%;
+  transform: translateX(-50%);
+
+  pointer-events: none;
+`
+
+const PosedContentContainer = posed(ContentContainer)({
+  init: { top: '-10px' },
+  copied: { top: '-75px', opacity: 0.5 },
+})
+
+// When `trigger()` is called, render copied content that floats up, then
+// disappears.
+export default class CopiedContentIndicator extends React.Component {
+  state = {
+    isActive: false,
+  }
+
+  trigger() {
+    this.setState({ isActive: true })
+  }
+
+  handlePoseComplete = () => {
+    this.setState({ isActive: false })
+  }
+
+  render() {
+    return (
+      <ContentAnchor>
+        {this.state.isActive && (
```

Comment:
I think this component should stay in the DOM when it is inactive. The current solution slows down the rendering performance. (https://developers.google.com/web/fundamentals/performance/rendering/stick-to-compositor-only-properties-and-manage-layer-count)
A better solution would be to change the `opacity` instead of removing the element. Additionally, a `will-change: opacity, transform;` would optimize the performance, but is not mandatory.

Reply:
I think I've fixed the first part of this. Re the CSS prop, should it be `will-change: opacity: top;`?

Reply:
Yes, it didn't run perfectly smooth as does now. I then checked it with the Chrome Dev Tools.
If you want to dig deeper into this topic, check out https://developers.google.com/web/fundamentals/. They've got some great articles out there.